ciao-storage/block.go

```diff
@@ -25,7 +25,9 @@ var (
 
 // BlockDriver is the interface that all block drivers must implement.
 type BlockDriver interface {
-	CreateBlockDevice(image *string, sizeGB int) (BlockDevice, error)
+	CreateBlockDevice(string, *string, int) (BlockDevice, error)
```

Comment:
It's useful self-documenting style to keep the human readable variable name, in my opinion.